src/ralph/parsers.py

```diff
@@ -40,29 +39,55 @@ def parse(self, input_file):
         """Parses GELF formatted logs (one JSON string event per row).
 
         Args:
-            input_file (string): Path to the log file to parse (could be gunzipped).
+            input_file (file-like): The log file to parse.
 
         Yields:
-            event: events raw short_message string
-
+            event: events raw short_message string.
         """
+
         logger.info("Parsing: %s", input_file)
 
         for event in input_file:
             try:
                 yield json.loads(event)["short_message"]
             except (json.JSONDecodeError, TypeError) as err:
-                logger.error(
-                    "Input event '%s' is not a valid JSON string! It will be ignored.",
-                    event,
-                )
+                msg = "Input event '%s' is not a valid JSON string! It will be ignored."
```

Comment:
```suggestion
                msg = (
                    "Input event '%s' is not a valid JSON string!\n"
                    "It will be ignored."
                )
```